rdm/render.py

```diff
@@ -34,30 +34,23 @@ def join_to(foreign_keys, table, primary_key='id'):
     return joined
 
 
-def md_indent(snippet, header_shift=0):
+def md_indent(snippet, header_shift = 0):
     lines = snippet.split('\n')
-    i = 0
     in_code = False
-    if header_shift < 0:
-        while i > header_shift:
-            i -= 1
-            for j in range(0, len(lines)):
-                if lines[j][0:3] == "```":
-                    in_code = not in_code
-                if in_code:
-                    continue
-                if lines[j][0] == '#':
-                    lines[j] = lines[j][1:].lstrip()
-    else:
-        while i < header_shift:
-            i += 1
-            for j in range(0, len(lines)):
-                if lines[j][0:3] == "```":
-                    in_code = not in_code
-                if in_code:
-                    continue
-                if lines[j][0] == '#':
-                    lines[j] = '#' + lines[j]
```

Comment:
Nice job shortening this down. Also, good commit message! It reads like you're following https://chris.beams.io/posts/git-commit/ (which I probably sent you during your coding challenge).